api/spec/v1.yaml

```diff
@@ -1874,11 +1881,28 @@ components:
           description: The public key identifying this Validator's node.
           example: *node_id_1
         escrow:
-          allOf: [$ref: '#/components/schemas/TextBigInt']
-          description: The amount staked.
+          allOf: [$ref: '#/components/schemas/Escrow']
+          description: The escrow account data for this validator.
+        voting_power:
+          type: integer
+          format: int64
+          description: The voting power of this Validator.
+        voting_power_total:
+          type: integer
+          format: int64
+          description: The total voting power across all Validators.
         active:
           type: boolean
           description: Whether the entity is part of validator set (top <scheduler.params.max_validators> by stake).
+        start_date:
+          type: string
+          format: date-time
+          description: The second-granular consensus time.
+          example: *iso_timestamp_3
+        rank:
+          type: integer
+          format: uint64
+          description: The rank of the Validator, determined by voting power.
         status:
```

Comment:
Unrelated to PR, but quick fix: Can we rename this pls? The name is completely uninformative in combination with the bool value. Maybe `is_active`, in line with Peter's https://github.com/oasisprotocol/nexus/pull/686/files#r1637773395 ?

Reply:
I started by changing `status` to `is_eligible` instead since we have an `active` field that denotes whether the validator is actively participating (ie whether it's in the top 120). I wanted `is_eligible` to convey whether the entity has an eligible validator node at all, but upon further investigation it turns out that we only have 124 entities with a validator node, which is less than the [226](https://www.oasisscan.com/validators) displayed on oasisscan. I suspect that oasisscan is displaying every entity that ever had a validator node, but I'll need to do more research on what constitutes active/waiting/inactive for a validator. 

Perhaps we only have 4 'waiting' validators and the rest can be considered inactive. At any rate I'll need to update the validators query because it makes some incorrect assumptions about the presence of a node.

I'll likely rename `is_eligible` to `is_waiting` or something similar, but I'll hold off until I'm sure the terminology is correct.

Reply:
I think there re 3 cases
- active (registered and running)
- "waiting" (registered and running but not in top 120 by stake, or it can also happen that the node just registered and is among top 120, but is not yet eligible - validator needs to be online for 2 epochs before it's picked up in the validator set)
- inactive (an entity that was a validator in past, but currently is not)

So, I think `active` makes sense when discriminating between validators who are online and those who are not. The second one should maybe something in line of `in_committe`/`is_validating`/`in_validator_set`, which would represent whether the validator is in the current validator set committee or not.

Reply:
Got it, thanks for the explanation! Updated to `active` and `in_validator_set`. One complication - we don't have a great way currently to distinguish between entities that are inactive validators and entities that have never been a validator since we don't yet store validator history. Once #703 is in, we can use that history to find inactive validators but for now I'll leave this endpoint as-is to only return active validators. Will file a ticket for the follow-up work there.

Reply:
> One complication - we don't have a great way currently to distinguish between entities that are inactive validators and entities that have never been a validator

👍  I think that's not that problematic/fine for now.

Reply:
Filed a follow-up ticket: https://app.clickup.com/t/86952ac6h